autode/neb/original.py

```diff
@@ -517,6 +517,9 @@ def from_end_points(
                 "Cannot construct a NEB from species with different atoms"
             )
 
+        if num < 2:
+            raise ValueError("Cannot create a NEB with fewer than 3 images")
```

Comment:
Shouldn't this be `num <= 2` if fewer than 3 images?

Reply:
ugh, no, didn't update the exception
```suggestion
            raise ValueError("Cannot create a NEB with fewer than 2 images")
```